tools/replay/parse_dump_resources_cli.cpp

```diff
@@ -330,303 +457,103 @@ bool parse_dump_resources_arg(gfxrecon::decode::VulkanReplayOptions& vulkan_repl
         return true;
     }
 
+    // Vulkan dump resources expectes only a json file
```

Comment:
The variable parse_error is set to false at the start of this method, never changed, and the value is compared at the end of this method. Should probably not declare or use it.

Reply:
fixed